bower-publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
+
+if [ $CURRENT_BRANCH != 'master' ] ; then
+  exit 0
```

Comment:
It’d be nice if the script always output something — maybe “This build isn’t on master. Skipping bower-chosen release.” here?